Automattic/vip-go-mu-plugins
# Jetpack: Update 10.9-beta2 to 10.9

## Description
This just updates the current Jetpack 10.9 beta2 version to 10.9.

## Changelog Description

### Plugin Updated: Jetpack 10.9

We upgraded to Jetpack 10.9 from 10.9-beta2

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
Check out PR.
[Pin Jetpack to 10.9](https://docs.wpvip.com/how-tos/test-jetpack-on-vip-go/#h-pinning-to-a-version)
Go to wp-admin > Jetpack
Verify Jetpack version is 10.9
